src/content/core/endpoints/automatic-ssl.mdx

```diff
@@ -8,6 +8,9 @@ id: automatic-ssl
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+Okteto streamlines the development process by automatically adding endpoints to the CLI and UI for services that are running.
+Simply add the specified annotation to your service's manifest, and Okteto will automatically handle the endpoint creation.
```

Comment:
Can we add a link somewhere in this page to the troubleshooting section?

Reply:
I'd like to avoid pushing people to troubleshooting from the introduction of this section. Is there another place where you think they would fit?